sdk/eventhubs/azure-messaging-eventhubs-track2-perf/src/main/java/com/azure/messaging/eventhubs/perf/EventProcessorJedisTest.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.messaging.eventhubs.perf;
+
+import com.azure.messaging.eventhubs.EventProcessorClient;
+import com.azure.messaging.eventhubs.EventProcessorClientBuilder;
+import com.azure.messaging.eventhubs.checkpointstore.jedis.JedisRedisCheckpointStore;
+import com.azure.messaging.eventhubs.models.ErrorContext;
+import com.azure.messaging.eventhubs.models.EventContext;
+import com.azure.messaging.eventhubs.models.EventPosition;
+import reactor.core.publisher.Mono;
+import redis.clients.jedis.JedisPool;
+import com.azure.perf.test.core.EventPerfTest;
+import redis.clients.jedis.JedisPoolConfig;
+import redis.clients.jedis.Protocol;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Consumer;
+
+
+public class EventProcessorJedisTest extends EventPerfTest<EventProcessorJedisOptions> {
+
+    private EventProcessorClient eventProcessorClient;
+    /**
+     * Creates an instance of performance test.
+     *
+     * @param options the options configured for the test.
+     * @throws IllegalStateException if SSL context cannot be created.
+     */
+    public EventProcessorJedisTest(EventProcessorJedisOptions options) {
+        super(options);
+        Duration errorAfter = options.getErrorAfterInSeconds() > 0
+            ? Duration.ofSeconds(options.getErrorAfterInSeconds()) : null;
+
+        JedisPoolConfig poolConfig = new JedisPoolConfig();
+        JedisPool jedisPool = new JedisPool(poolConfig, options.getHostName(), 6380, 5000, 1000, options.getPassword(), Protocol.DEFAULT_DATABASE, options.getUserName(), true, null, null, null);
+        JedisRedisCheckpointStore checkpointStore = new JedisRedisCheckpointStore(jedisPool);
+
+        Consumer<ErrorContext> errorProcessor = errorContext -> super.errorRaised(errorContext.getThrowable());
+        Consumer<EventContext> eventProcessor = eventContext -> {
+            super.eventRaised();
+            eventContext.updateCheckpoint();
+        };
+
+        Map<String, EventPosition> initalPositionMap = new HashMap<>();
+        for(int i = 0; i<options.getPartitions(); i++){
```

Comment:
Format this. Ctrl+Alt+L.

Reply:
Do we need this parameter? You can query the Event Hub for this information.  (ie. client.getEventHubProperties())